src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -326,10 +357,26 @@ public void validate(FailureCollector collector) {
         }
       }
 
+      if (containsMacro(NAME_CONTENT_TYPE)) {
```

Comment:
no need for this check, by default if its a macro or not provided at configure time, it will be null

Reply:
PR Updated.